apps/fluent-tester/src/TestComponents/Icon/IconExperimental.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { FontIcon, SvgIcon, Icon } from '@fluentui-react-native/experimental-icon';
+import { Text, View } from 'react-native';
+import TestSvg from './assets/test.svg';
+import { SvgIconProps } from '@fluentui-react-native/icon';
+
+const fontBuiltInProps = {
+  fontFamily: 'Arial',
+  codepoint: 0x2663,
+  fontSize: 32,
+};
+
+export const IconExperimental: React.FunctionComponent = () => {
+  const svgProps: SvgIconProps = {
+    uri: 'https://upload.wikimedia.org/wikipedia/commons/8/84/Example.svg',
+    viewBox: '0 0 1000 1000',
+  };
+  return (
+    <View>
+      <Text>Icon component</Text>
+      <Icon svgSource={{ width: 100, height: 100, color: 'purple', viewBox: '0 0 500 500', src: TestSvg }} />
```

Comment:
Do we need to gate this for windows?

Probably it'd actually be better if the Icon file is changed to not render anything for SVGs on the windows endpoint instead of adding platform checks in the test everywhere

Reply:
I feel like it should be both: Update Icon to throw a warning / error if you use SVGs on windows, but also do the platform checks in our test app to make it explicit in our "example" code that is frequently linked to partners this isn't supported. I don't have too strong of an opinion though.

Reply:
I would add this check inside the component first. And some sort of error/warning. I don't think we'll need it in the Tester for the new Icon because we plan to support it when V1 Icon is finished.

Reply:
I added check for Platform inside the component. Should I do the same in old Icon component? Or I can add for this a separate task. We can have a check inside the component and message which says that currently SVG Icons are not supported on Windows

Reply:
Chatted offline but yeah, doing that probably makes sense since right now it probably just crashes.